libzfs-sys/build.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2017 Intel Corporation. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+extern crate bindgen;
+extern crate pkg_config;
+
+use std::env;
+use std::path::PathBuf;
+
+fn main() {
+    env::set_var("LIBCLANG_PATH", "/opt/llvm-5.0.0/lib64/");
+
+    pkg_config::Config::new().probe("libzfs").unwrap();
+    println!("cargo:rustc-link-lib=zpool");
+
+
+    let bindings = bindgen::Builder::default()
+        .header("wrapper.h")
+        .opaque_type("libzfs_handle_t")
+        .hide_type("nvlist_t")
+        .hide_type("nvlist")
+        .constified_enum("boolean")
+        .whitelisted_function("libzfs_init")
+        .whitelisted_function("libzfs_fini")
+        .whitelisted_function("thread_init")
+        .whitelisted_function("thread_fini")
+        .whitelisted_function("zpool_import")
```

Comment:
do these allow for the use of "-a" in the commands for import and export?

Reply:
The import / export commands are actually a *lot* of code. 

I'll probably end up whitelisting a few more fns over time, but it's lot's of nvpair munging.